homeassistant/components/nws_alerts/config_flow.py

```diff
@@ -0,0 +1,144 @@
+"""Config flow for NWS Alerts integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import requests
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import API_ENDPOINT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("api_key"): str,
+        vol.Required("friendly_name", default="NWS Alerts"): str,
+        vol.Required("update_interval", default=90): int,
```

Comment:
Friendly name and update interval should be handled automatically by Home Assistant. There is no need to request them from the user.
```suggestion
```

Reply:
What if I wanted to recommend an update interval?